src/components/layout/seo.tsx

```diff
@@ -89,9 +89,9 @@ const SEO = ({
 
   const metaDescription = description || t('main.description');
   const typeOfSite = siteType || 'website';
-  const defaultImageUrl =
-    site.siteMetadata.siteUrl + DEFAULT_META_IMAGE_URL_PATH;
-  const metaImageUrl = imageUrl ?? defaultImageUrl;
+
+  const metaImageUrl =
```

Comment:
every image we pass in need to be path. never an url as seo should resolve that. which we do now.